### DISCUSSION TITLE
Make Dasharo Tool Suite available on netboot.xyz

### DISCUSSION THREAD

**Discussion by Firminator** [Dasharo/docs#274] [Ideas]
https://docs.dasharo.com/common-coreboot-docs/dasharo_tools_suite/#bootable-over-network could also be made available in the official netboot.xyz menu. This could help adoption of Dasharo firmware for supported devices. Or people who have already Dashora hardware and a working netboot.xyz iPXE boot environment could easily checkout and test DTS without messing around with USB flash drives.
Basically ask Anthony @https://github.com/netbootxyz/netboot.xyz/discussions if he can add https://3mdeb.com/open-source-firmware/DTS/v1.0.0/dts-base-image-ce-v1.0.0.wic.gz to the netboot.xyz menu and see what he says.

**Comment by pietrushnic** [comment]
@Firminator as always very good idea. We will definitely think about trying that with next release.
